### PR TITLE
Release/v6.11.0

### PR DESCRIPTION
**Pull request by nijotz**
## Proposed Release Notes

* Open-sourced agent under Apache 2.0 license
  * Added CODE_OF_CONDUCT.md file
  * Streamlined README.md file
  * Updated CONTRIBUTING.md file
  * Added additional guidance to bug report template
  * Added copyright headers to all source files
* Added Distributed Tracing option to config file used for first time customers
* Converted some test files to Node-tap
* Removed "hidden" and unused code injector diagnostic capability
* Upgraded @grpc/grpc-js from 1.0.4 to 1.0.5

### CODE REVIEW

**Comment by nijotz** [comment]
> These should probably all have release notes. Could leave off the tap test conversions but usually we still have notes on those too for contributors.
> 
>     * #409
> 
>     * #401
> 
>     * #395
> 
>     * #393
> 
> 
> Other than that, the note looks good. Although, maybe we should see if there's any recommended verbiage.

Got those release notes added as well

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
...
+  * Added CODE_OF_CONDUCT.md file
+  * Streamlined README.md file
+  * Updated CONTRIBUTING.md file
+  * Added additional guidance to bug report template
+  * Added copyright headers to all source files
+* Added Distributed Tracing option to config file used for first time customers
+* Converted some test files to Node-tap
+* Removed "hidden" and unused code injector diagnostic capability
+* Upgraded @grpc/grpc-js from 1.0.4 to 1.0.5
+* Upgraded escodegen from 1.14.1 to 1.14.2
```
Comment (michaelgoin) [inline NEWS.md:13]: This PR did not land (closed without merging) and the dependency was removed as a part of removing the code injector.

```suggestion
```

File: NEWS.md [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
+* Upgraded @newrelic/aws-sdk from 1.1.2 to 1.1.3
```
Comment (michaelgoin) [inline NEWS.md:14]: This PR did not land (closed without merging).

```suggestion
```

File: NEWS.md (hunk 1 above)
Comment (nijotz) [reply NEWS.md]: Fixed

File: NEWS.md (hunk 2 above)
Comment (nijotz) [reply NEWS.md]: Fixed

**Comment by nijotz** [comment]
Manually merged this PR into main using CLI

**Pull request closed without merging**